cmd/e2e/gameserverapi_test.go

```diff
@@ -139,6 +140,9 @@ var _ = Describe("GameServerAPI tests", func() {
 			err = json.Unmarshal(body, &gs)
 			g.Expect(err).ToNot(HaveOccurred())
 			g.Expect(gs.Name).To(Equal(gsName))
+			// check that GameServers have a NodeName and a PublicIP
+			g.Expect(gs.Status.NodeName).ToNot(BeEmpty())
```

Comment:
any chance we can add that check to other e2e tests as well (at least one)?
I'm thinking that maybe if in the future we remove the gameserverapi from the main Thundernetes repo, this code will be removed as well so there won't be any tests for this functionality